src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -389,7 +374,7 @@ class UworkerOutput(UworkerMsg):
 
   def save_rich_type(self, attribute, value):
     field = getattr(self.proto, attribute)
-    if isinstance(value, dict):
```

Comment:
I don't think we should make this change, it's not obvious to me that a list should always be a json field in every case. Maybe keep the override method in fuzz and leave this unchanged.

Reply:
which lists should be json fields and which should not?